src/App.tsx

```diff
@@ -57,6 +58,7 @@ function App() {
           <Route exact path="/login" component={Login} />
           <Route exact path="/signup" component={Signup} />
           <Route exact path="/register" component={SignupForm} />
+          <Route exact path="/exit" component={ExitSurveyModal} />
```

Comment:
By definition, a modal pops up on an existing page like the home page, it shouldn't be going to it's own page.